server/static/js/index/home_page_modal_windows.js

```diff
@@ -0,0 +1,298 @@
+ */
+function resetRowSelection () {
+    $(TABLE_ROW).click(function () {
+        $activeRow = $(this);
+        $(TABLE_ROW).removeClass("row-active");
+        $activeRow.addClass("row-active");
+        $('[data-modal="edit"]').addClass("btn-active");
+        $('[data-modal="debug"]').addClass("btn-active");
+        rowSelected = true;
+    });
+}
+
+/**
+ * The function sets the values of the modal window fields in accordance with the selected line (for devices)
+ */
+function setDeviceModalWindowValues () {
+    let row = $activeRow;
+    let $deviseType = row.children('[data-device-type]').attr('data-device-type');
+    let $deviceID = row.children('[data-device-type-id]').text();
+    let $deviseIdHidden = row.children('[data-device-id]').val();
+
+    $(MODAL_DEVICE_ID_HIDDEN).val($deviseIdHidden);
+    $(MODAL_DEVICE_ID).val($deviceID);
+    $(MODAL_DEVICE_TYPE).val($deviseType);
+}
+
+/**
+ * The function of forming a JSON object by reading the data of the modal window
+ * @returns {{deviceData: {}}}
+ */
+function getDevicePayload () {
+    let payload = {"deviceData": {}};
+
+    payload.deviceData.id = $(MODAL_DEVICE_ID_HIDDEN).val();
+    payload.deviceData.device_id = $(MODAL_DEVICE_ID).val();
+    payload.deviceData.device_type = $(MODAL_DEVICE_TYPE).val();
+    payload.deviceData.device_access_key = $(MODAL_DEVICE_KEY).val();
+
+    return payload;
+}
+
+/**
+ * The function sets the values of the modal window fields in accordance with the selected line (for users)
+ */
+function setUserModalWindowValues () {
+    let row = $activeRow;
+    let $deviceIdHidden = row.children('[data-user-id]').val();
+    let $userFirstName = row.children('[data-user-name]').attr('data-first-name');
+    let $userLastName = row.children('[data-user-name]').attr('data-last-name');
+    let $userEmail = row.children('[data-user-email]').text();
+    let $userActive = row.children('[data-user-enabled]').attr('data-user-enabled');
+    let $hiddenContainer = row.children('td:last-child').children('.user-roles-container').children('input');
+    let userRolesList = [];
+
+    $(MODAL_USER_ID_HIDDEN).val($deviceIdHidden);
+    $(MODAL_USER_FIRST_NAME).val($userFirstName);
+    $(MODAL_USER_LAST_NAME).val($userLastName);
+    $(MODAL_USER_EMAIL).val($userEmail);
+
+    if ($userActive === 'true') {
+        $('#modal_user_active').prop("checked", true);
+    }
+
+    $hiddenContainer.each(function () {
+        userRolesList.push($(this).val());
+        for (let userRoleIndex = 0; userRoleIndex < userRolesList.length; userRoleIndex++) {
+            let userRole = userRolesList[userRoleIndex];
+            $(MODAL_USER_CHECKBOX).each(function () {
+                if ($(this).val() === userRole) {
+                    $(this).prop("checked", true);
+                }
+            });
+        }
+    });
+}
+
+/**
+ * The function of forming a JSON object by reading the data of the modal window
+ * @return {{userData: {roles: Array}}}
```

Comment:
Same here - Incorrect return value. We do not describe entire object - we simply specify what is it and what's it for.